backend/solver/origuide/fold/fold.py

```diff
@@ -6,9 +6,9 @@ def is_steady_state(frame):
 
 
 def extract_root_frame(root):
-    return {k: v for k, v in root.items() if not k.startswith('file_')}
-
-
+    frame = {k: v for k, v in root.items() if not k.startswith('file_')}
+    return frame if frame == {} else None
```

Comment:
Waaait, what?

Reply:
If the fold's root does not contain any frame related properties we don't have a frame in the root and assume all the frames are in the `file_frames` property. This is not necessary anymore after recent changes to the frontend.

Left it like this,  because it sounds like a feature. ;)

Reply:
Waaait, what?